addons/interaction/functions/fnc_canInteractWithVehicleCrew.sqf

```diff
@@ -0,0 +1,26 @@
+#include "..\script_component.hpp"
+/*
+ * Author: johnb43
+ * Checks if a unit can interact with the vehicle crew inside.
+ *
+ * Arguments:
+ * 0: Player <OBJECT>
+ * 1: Vehicle <OBJECT>
+ *
+ * Return Value:
+ * Unit can interact with vehicle crew <BOOL>
+ *
+ * Example:
+ * [cursorObject, player] call ace_interaction_fnc_canInteractWithVehicleCrew
+ *
+ * Public: No
+ */
+
+params ["_player", "_vehicle"];
+
+private _sidePlayer = side _player;
```

Comment:
`side group _player`?

Reply:
I figured we would never need to know the real side of the player if you are using this function. However I don't think it hurts and it's safer.